### DISCUSSION TITLE
Cloud tech radar main tag

### DISCUSSION THREAD

**Discussion by iwoak** [pagopa/technology-radar#26] [General]
As mantainers of cloud tech radar we need to choice a main identifier tag. The first one was 'cloud' but in the last PR we are using DevOps for a tool, @uolter I vote for DevOps. What do you think about? @pasqualedevita @Krusty93

**Comment by uolter** [comment]
> As mantainers of cloud tech radar we need to choice a main identifier tag. The first one was 'cloud' but in the last PR we are using DevOps for a tool, @uolter I vote for DevOps. What do you think about? @pasqualedevita @Krusty93

I really don't know. In this case is more devops not necessarily cloud. Anyway I have no prevailing opinion. You do you.

**Comment by krusty93** [comment]
@iwoak Are you referring to [these ones](https://github.com/pagopa/technology-radar/pull/25/files#diff-dce07c49d9b28f5600e489ca8021d15c0cf913066dd7513a6365aa4d30ccabb5R5)?

In affirmative case, why don't we use both of them? i.e.:
- Terraform: both `cloud` and `devops`
- Azure AppService: `cloud`
- GitHub actions: `devops`

EDIT: I've found [this PR](https://github.com/pagopa/technology-radar/pull/14/files) from @uolter which contains both

**Reply by iwoak** [reply]
yep @Krusty93, we spoken about it and we decided to use always `cloud` as first tag for every entry talking about infrastracture.
The others tag would be used to describe better the card content.
